cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/BulkSparkConf.java

```diff
@@ -147,9 +143,7 @@ public BulkSparkConf(SparkConf conf, Map<String, String> options)
         this.consistencyLevel = ConsistencyLevel.CL.valueOf(MapUtils.getOrDefault(options, WriterOptions.BULK_WRITER_CL.name(), "EACH_QUORUM"));
         this.localDC = MapUtils.getOrDefault(options, WriterOptions.LOCAL_DC.name(), null);
         this.numberSplits = MapUtils.getInt(options, WriterOptions.NUMBER_SPLITS.name(), DEFAULT_NUM_SPLITS, "number of splits");
-        this.rowBufferMode = MapUtils.getEnumOption(options, WriterOptions.ROW_BUFFER_MODE.name(), DEFAULT_ROW_BUFFER_MODE, "row buffering mode");
-        this.sstableDataSizeInMB = MapUtils.getInt(options, WriterOptions.SSTABLE_DATA_SIZE_IN_MB.name(), 160, "sstable data size in MB");
-        this.sstableBatchSize = MapUtils.getInt(options, WriterOptions.BATCH_SIZE.name(), 1_000_000, "sstable batch size");
+        this.sstableDataSizeInMB = MapUtils.getInt(options, WriterOptions.SSTABLE_DATA_SIZE_IN_MB.name(), DEFAULT_SSTABLE_DATA_SIZE_IN_MB, "sstable data size in mebibytes");
```

Comment:
if data size is mebibytes, we should probably rename the variables to MIB.

```
WriterOptions.SSTABLE_DATA_SIZE_IN_MB -> WriterOptions.SSTABLE_DATA_SIZE_IN_MIB
DEFAULT_SSTABLE_DATA_SIZE_IN_MB -> MIB
```

Reply:
I am with you. The reason that I only changed the description is because the option has been named as "MB" but used as "MiB" since it is introduced. 
How about I add a new option, mark this one `WriterOptions.SSTABLE_DATA_SIZE_IN_MB` as deprecated (with the new description as the correction), and both are treated the same, as they are both in MiB effectively.

Reply:
+1 that makes sense